### DISCUSSION TITLE
tables behaving weirdly attempting to scroll using mouse wheel

### DISCUSSION THREAD

**Discussion by tim-wilkins** [primefaces/community#1685] [PrimeNG]
With the latest version of primeng (17.13.0) I've had a problem whereby if I try to scroll down it keeps jumping back.  This is similar to a another problem I've hit in the past with virtual scrolling  but this doesn't affect scrolling by dragging the thumb, which is fine, but it does affect clicking the arrows at the top and bottom (of a vertical scrollbar).  The affected tables do use virtual scrolling but I don't know if that's significant (most of my tables do) as yet and some similar tables aren't affected for no apparent reason!  For the moment reverting to 17.12.0 fixes the problem.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by tim-wilkins** [comment]
I can't share the actual application - it's too large and relies upon a number of other applications, a database, etc.  I'll see if I can get a separate application to show the problem, but it may be a little while before I'm able to do this (other development priorities) and I don't yet know the circumstances when this occurs.

**Comment by tim-wilkins** [comment]
I've tried a simple application without success in reproducing the problem.

**Comment by tim-wilkins** [comment]
More observations:  The html has switch statements to format different types of data as well as loops but I've modified the code to use the newer style (e.g. @switch rather than ngSwitch) which is apparently faster and this improves things.  At least some of the problem tables are within tab controls and switching to another tab and back now seems to cure the problem.  Perhaps some sort of timing issue?

**Comment by tim-wilkins** [comment]
This seems to be fixed in 17.14.1 (probably 17.14.0 but I didn't try that).

**Comment by cetincakiroglu** [comment]
Hi @tim-wilkins,

We need an example to investigate and identify the issue, could you please share a stackblitz by the easiest way possible so we can better understand what's wrong?

**Comment by tim-wilkins** [comment]
I wasn't table to generate an example apart from within a set of applications and this is no longer a problem with the latest release of PrimeNg.
